app/components/score-detail.js

```diff
@@ -0,0 +1,10 @@
+import Component from '@ember/component';
+
+export default Component.extend({
```

Comment:
Looks like the template for this component still needs `latestReview` and `isLatestReleaseInLast3Months` and `hasGithubData`